crawlerr/services/rag_service.py

```
@@ -341,6 +586,86 @@ async def _rerank_results(
+            loop = asyncio.get_event_loop()
+            
+            scores = []
+            for pair in pairs:
+                score = await loop.run_in_executor(
+                    None, 
+                    lambda p=pair: self.reranker.predict([p])[0]
+                )
+                scores.append(score)
+            
+            # Update match scores with reranker predictions
+            for match, score in zip(matches, scores):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)